internal/bodyprocessors/json_test.go

```diff
@@ -116,18 +124,49 @@ var jsonTests = []struct {
 			"json.1.f.0.0":     "1",
 			"json.1.f.0.0.0.z": "abc",
 		},
+		err: nil,
+	},
+	{
+		name: "broken1",
+		json: `{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a":{"a": 1 }}}}}}}}}}}}}}}}}}}}}}`,
+		want: map[string]string{},
+		err:  errors.New("invalid JSON"),
```

Comment:
wonder why is this invalid. Can we specify in the name?

Reply:
Oh, there are more open brackets than closing.